### PR TITLE
10.selectors

### PR DESCRIPTION
**Pull request by cardel**

### CODE REVIEW

File: cypress/page/address-step.page.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ class AddressStepPage {
   private content: string;
 
   constructor() {
-      this.carNav = ".cart_navigation";
+      this.carNav = ".cart_navigation .button span";
```
Comment (holgiosalos) [inline cypress/page/address-step.page.ts]: you don't need to change this selector, as you are using the get in combination with the contains

File: cypress/page/shipping-step.page.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ class ShippingStepPage {
 
   constructor() {
       this.cartContent = "Proceed to checkout";
-      this.cartNav = ".cart_navigation";
+      this.cartNav = ".cart_navigation .button span";
```
Comment (holgiosalos) [inline cypress/page/shipping-step.page.ts]: you don't need to change this selector, as you are using the get in combination with the contains

File: cypress/page/address-step.page.ts (hunk 1 above)
Comment (cardel) [reply cypress/page/address-step.page.ts]: Fixed

File: cypress/page/shipping-step.page.ts (hunk 2 above)
Comment (cardel) [reply cypress/page/shipping-step.page.ts]: Fixed

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**